mypy/util.py

```diff
@@ -467,30 +474,63 @@ class FancyFormatter:
     def __init__(self, f_out: IO[str], f_err: IO[str], show_error_codes: bool) -> None:
         self.show_error_codes = show_error_codes
         # Check if we are in a human-facing terminal on a supported platform.
-        if sys.platform not in ('linux', 'darwin'):
+        if sys.platform not in ('linux', 'darwin', 'win32'):
             self.dummy_term = True
             return
         force_color = int(os.getenv('MYPY_FORCE_COLOR', '0'))
         if not force_color and (not f_out.isatty() or not f_err.isatty()):
             self.dummy_term = True
             return
-
+        if sys.platform == 'win32':
+            self.dummy_term = self.initialize_win_colors()
```

Comment:
I would rather make the bool value opposite for both methods and document them as "Return True if initialization was successful, False otherwise".